host_vars/sarvar.yml

```diff
@@ -44,3 +44,7 @@ scrape_config:
     static_configs:
       - targets:
           - 'localhost:9187'
+  - job_name: '1c_exporter'
+    static_configs:
+      - targets:
+         - 'localhost:{{ 1c_exporter_port }}
```

Comment:
metrics_path ?

Reply:
node_exporter does not need one, why for 1c_exporter?

Reply:
![image](https://user-images.githubusercontent.com/13658971/147077433-008a75bb-a9e8-4790-8368-d125656c0bc6.png)